serverless/knative-serving/external-ingress-routing/using-http2-gRPC.adoc

```diff
@@ -7,3 +7,10 @@ include::_attributes/common-attributes.adoc[]
 {ServerlessProductName} supports only insecure or edge-terminated routes. Insecure or edge-terminated routes do not support HTTP2 on {product-title}. These routes also do not support gRPC because gRPC is transported by HTTP2. If you use these protocols in your application, you must call the application using the ingress gateway directly. To do this you must find the ingress gateway's public address and the application's specific host.
 
 include::modules/interacting-serverless-apps-http2-gRPC.adoc[leveloffset=+1]
+
+[id="additional-resources_serverless-using-http2-gRPC"]
+[role="_additional-resources"]
+== Additional resources
```

Comment:
Is this additional resources section meant to apply to the whole assembly, or just the `interacting-serverless-apps-http2-gRPC.adoc` module? If it applies to the whole assembly, I would move it to the bottom. If it is meant to go with the first module, it should be changed to `.Additional resources`. 

(For `.Additional resources` headings, you don't need the ID - unless there's an xref somewhere that will break without it - but you do need the `[role="_additional-resources"]` statement.)